src/server/string_family.cc

```diff
@@ -693,49 +696,50 @@ void SetCmd::RecordJournal(const SetParams& params, string_view key, string_view
 }
 
 void StringFamily::Set(CmdArgList args, ConnectionContext* cntx) {
-  string_view key = ArgS(args, 0);
-  string_view value = ArgS(args, 1);
+  facade::CmdArgParser parser{args};
+
+  string_view key = parser.Next();
+  string_view value = parser.Next();
```

Comment:
you can parse 2 elements at once if you want
auto [key, value] = parser.Next<sting_view, string_view>();